project.sh

```diff
@@ -562,11 +562,17 @@ local-clean(){
 -test-clj () {
 	allow-snapshots
 	local cmd
+	#TODO make args order independent
 	case $1 in
 		-r|--refresh|--watch)
 			cmd='--watch'
 			shift;;
 	esac
+	case $1 in
+		-ff|--fail-fast)
```

Comment:
Friendly fire

Reply:
What if you pass the order `-ff -r`?

Reply:
I won't pick up the `-r`. Thus the TODO.
Suggestions welcome

Reply:
```
-test-clj () {
        allow-snapshots
	local cmd
	local remaining
	while [[ -n "$1" ]];do
		case "$1" in
			-r|--refresh|--watch)
				cmd="$cmd --watch";;
			-ff|--fail-fast)
				cmd="$cmd --fail-fast";;
			*)
				remaining="$remaining $1";;
		esac
		shift
	done
	cmd=$(echo "$cmd" | xargs)
	remaining=$(echo "$remaining" | xargs)
	if [ -n "$remaining" ];then
		cmd="$cmd --focus $remaining"
	fi
	...
}
```